ZzaDashboard/RelayCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Input;
+
+
+namespace ZzaDashboard
+{
+    public class RelayCommand : ICommand
```

Comment:
Belongs to ViewModel layer, so should be moved to the appropriate folder in VewModel.